python/GafferSceneTest/ShaderQueryTest.py

```diff
@@ -128,16 +128,16 @@ def checkChildrenCount( plug, count ) :
 
 		a = q.addQuery( Gaffer.IntPlug() )
 		checkChildrenCount( q, 1 )
-		self.assertEqual( q["queries"]["query1"]["name"].getValue(), "" )
-		self.assertEqual( q["queries"]["query1"]["value"].getValue(), 0 )
+		self.assertEqual( q["queries"]["query0"]["name"].getValue(), "" )
+		self.assertEqual( q["queries"]["query0"]["value"].getValue(), 0 )
```

Comment:
Could we replace `["queryN"]` with `[N]` in the unit tests, to reflect our intention to require integer-based (rather than name-based) indexing for ArrayPlugs in future?

Reply:
Fixed in 4498f32d92920b24a3bd7e15dbc435d78aa40767